server/WebAPI/DTO/PaginationRequestDTO.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace WebAPI.DTO
+{
+  public class PaginationRequestDTO
+  {
+
+    [Required]
+    [Range(0, Int32.MaxValue)]
+    public int? PageNumber { get; set; }
+
+    [Required]
+    [Range(1, 24)]
```

Comment:
why not int.max? why 24?)

Reply:
cause i don't need to get 2 147 483 647 results for one time

Reply:
24 - magic number )

Reply:
Well, I consider that 24 is enough for the maximum number of results